cylc/flow/job_file.py

```diff
@@ -83,7 +83,7 @@ def write(self, local_job_file_path, job_conf, check_syntax=True):
         if check_syntax:
             try:
                 with Popen(
-                        ['/bin/bash', '-n', tmp_name],
+                        ['/usr/bin/env', 'bash', '-n', tmp_name],
```

Comment:
This file needs reviewing.

Reply:
It seems that the `/usr/bin/env` pattern is pretty well supported, we've been using it for Python for a while now without issue.